ci-operator/config/openshift/release/openshift-release-master__nightly-4.8.yaml

```diff
@@ -350,6 +350,11 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-csi
+- as: e2e-aws-csi-migration
+  interval: 12h
```

Comment:
I was wondering if 12 hours isn't too frequent, given that most non-CSI periodic jobs have an interval of 48 hours.

Reply:
If I understood the system correctly, most of the other jobs are run as part of nightly builds + tests that are run much more often (few times per day, not sure why we call them nightly). This job is not mature yet to get there, but once it is part of nightly, we should indeed reduce the frequency here.